src/Agent.Sdk/SecretMasking/OssSecretMasker.cs

```diff
@@ -13,6 +15,28 @@ namespace Agent.Sdk.SecretMasking;
 public sealed class OssSecretMasker : ISecretMasker, IDisposable
 {
     private SecretMasker _secretMasker;
+    private Telemetry _telemetry;
+
+    /// <summary>
+    /// The maximum number properties to report in a single
+    /// 'SecretMaskerDetections' telemetry event. This value was chosen to keep
+    /// the size in the range of existing routine telemetry events.
```

Comment:
>routine [](http://example.com/codeflow?start=42&length=7)

Can't resist asking what do you mean by 'routine' here? :) This sentence reads sensibly without that qualifier. Do you just mean `typical`? SEV5, just askin'

Reply:
Yeah, I mean typical. :)